generator.rb

```diff
@@ -0,0 +1,73 @@
+# Initiate array of vowels accessible everywhere
+VOWELS = ["a", "e", "i", "o", "u"]
+
+# Check if character is a vowel
+def is_vowel? (letter)
+    if VOWELS.include? (letter)
+        true
+    else
+        false
+    end
+end
+
+# Get user input to form new word
+def run_generator
+
+  puts "What is the first word to combine?"
+  word_A = gets.chomp
+  until word_A.size >= 2 
+    puts "Your word should have at least 2 letters."
+    word_A = gets.chomp
```

Comment:
In Ruby, we typically name variables using all lowercases letters, so this would be `word_a`.